decaylanguage/dec/dec.py

```diff
@@ -155,6 +155,14 @@ def parse(self, include_ccdecays=True):
         # At last, find all particle decays defined in the .dec decay file ...
         self._find_parsed_decays()
 
+        # Check whether certain decay model parameters are defined via
+        # variable names with actual values provided via 'Define' statements,
+        # and perform the replacements name -> value where relevant.
+        # Do also a replacement of 'a_float' with a_float.
+        dict_define_defs = self.dict_definitions()
+        [DecayModelParamValueReplacement(define_defs=dict_define_defs).visit(tree)
+        for tree in self._parsed_decays]
```

Comment:
Please make this an explicit loop, rather than relying on the fact that it has to run through the loop to make the list. It's best to not use comprehensions if you don't want the output result. Plus it gives you nice indentation for the two lines for free. (Sometimes it is handy on the REPL to do it this way, since it can avoid an extra line)

Reply:
Sure.